components/terrain/quadtreeworld.cpp

```diff
@@ -400,21 +351,23 @@ void QuadTreeWorld::accept(osg::NodeVisitor &nv)
                 int x,y;
                 stream >> x;
                 stream >> y;
-                traverseToCell(mRootNode.get(), vd, x,y);
+                mRootNode->traverseTo(vd, 1, osg::Vec2f(x+0.5,y+0.5));
             }
             else
-                traverse(mRootNode.get(), vd, cv, mRootNode->getLodCallback(), cv->getViewPoint(), true, mViewDistance);
+                mRootNode->traverse(vd, cv->getViewPoint(), mLodCallback, mViewDistance);
         }
         else
-            mRootNode->traverse(nv);
-    }
-    else if (isCullVisitor)
-    {
-        // view point is the same, but must still update visible status in case the camera has rotated
-        for (unsigned int i=0; i<vd->getNumEntries(); ++i)
         {
-            ViewData::Entry& entry = vd->getEntry(i);
-            entry.set(entry.mNode, !static_cast<osgUtil::CullVisitor*>(&nv)->isCulled(entry.mNode->getBoundingBox()));
+            osgUtil::IntersectionVisitor* iv = static_cast<osgUtil::IntersectionVisitor*>(&nv);
+            osgUtil::LineSegmentIntersector* lineIntsersector = dynamic_cast<osgUtil::LineSegmentIntersector*>(iv->getIntersector());
```

Comment:
The typo here and later in the variable name should be corrected. "lineIntersector"